src/OpenApi/Config/Tags.php

```diff
@@ -152,6 +152,7 @@ enum Tags: string
     case Elements = 'Elements';
     case ExecutionEngine = 'Execution Engine';
     case Emails = 'E-Mails';
+    case Export = 'Export';
```

Comment:
Please also add a OpenApi Tag here, so the tags are then sorted by this.
Currently Export is at the bottom.

Reply:
https://github.com/pimcore/studio-backend-bundle/pull/709/commits/32e295308146c2b6ccf6987665016c539139d778